builtin/providers/aws/resource_aws_cloudformation_stack.go

```diff
@@ -69,10 +73,14 @@ func resourceAwsCloudFormationStack() *schema.Resource {
 				Computed: true,
 			},
 			"policy_body": &schema.Schema{
-				Type:      schema.TypeString,
-				Optional:  true,
-				Computed:  true,
-				StateFunc: normalizeJson,
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validateJsonString,
+				StateFunc: func(v interface{}) string {
+					json, _ := normalizeJsonString(v)
+					return json
```

Comment:
Probably a nit pick - but looking at the code makes me think we should have a tiny function for this which we could simply reference as `StateFunc` to avoid the repetition, but I frankly can't think of a good name for such function 💭 
and maybe the repetition of 2-3 lines isn't that bad after all.